assets/environment-interpreter/activate/activate.d/start.bash

```diff
@@ -1,111 +1,124 @@
-"$_flox_activate_tracer" "$_activate_d/start.bash" START
-
 _comm="@coreutils@/bin/comm"
 _daemonize="@daemonize@/bin/daemonize"
 _flox_activations="@flox_activations@"
 _sed="@gnused@/bin/sed"
 _sort="@coreutils@/bin/sort"
 
-# If interactive and a command has not been passed, this is an interactive
-# activate,
-# and we print a message to the user
-# If inside a container, FLOX_ENV_DESCRIPTION won't be set, and we don't need to
-# print a message
-if [ -t 1 ] && [ $# -eq 0 ] && [ -n "${FLOX_ENV_DESCRIPTION:-}" ]; then
-  echo "✅ You are now using the environment '$FLOX_ENV_DESCRIPTION'." >&2
-  echo "To stop using this environment, type 'exit'" >&2
-  echo >&2
-fi
+start() {
+  _flox_activation_state_dir="${1?}"
```

Comment:
Is this the correct expansion method to use if the parameter is optional? I think you get the wrong behavior if no argument was passed.

> If parameter is null or unset, the expansion of word (or a message to that effect if word is not present) is written to the standard error and the shell, if it is not interactive, exits. Otherwise, the value of parameter is substituted.

When I run `echo "'${foo?}'"` with `foo` unset I see this:

```
bash: foo: parameter not set
```

Reply:
Maybe you want `"${1:-}"` instead